### DISCUSSION TITLE
Help with creating my open source program

### DISCUSSION THREAD

**Discussion by arvaus** [community/community#146075] [Programming Help]
### Body

I’m new to developing software as intricate as I’m trying to develop. I beekeeper of five years. I have five beehives. I have looked around for beekeeping software that I like. I decided I would start my own open source software after seeing the success of jellyfin. The software that I found that I think has a lot of good features is really expensive and I only have five beehives so I don’t really need software that I have to pay for. I know of a lot of other people also have the same problem. The issue is that for the features that I would like is stuck behind a pay wall, so people who only have a few beehives for a hobby don’t want to pay for an expensive program. I decided to create an open source software project so that the software can become very helpful. My issue is that I only know some HTML, some Java, and a lot of python, but I don’t really think Python would be helpful in this situation. I also don’t know how to combine them to be able to work together into one nice HTML program that will save reports onto your computer. This is where you guys come in. I created a repository in GitHub, but I need your help. I don’t know how to do a lot of things so I would like to make this open source for anybody to help me with. Thank you for all of you who help me. I will leave the GitHub repository link at the end of this article.
PS: I know there’s not a lot there right now, but I will be adding a lot in the next coming days.
Link: https://github.com/arvaus/Bee-Heaven
**I would like to make it so that the program would save the entries in my HTML form and then the user could click a export button when they want to export the data and  it will download a excel spreadsheet with all of the past entries. I want it to export all of the past entries insted of just that days entry. I cant figure out how to do this though.**

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Aravindhan-KS** [comment]
I could contribute to the project , I currently have finals till 24th of December and I could start contributing from 6th Jan , also I don't have much knowledge in website development , my university is about to teach full stack development in the upcoming semester I could contribute more towards the front end of the website(CSS)

**Comment by ncast-dev** [comment]
## DIGI-338 — Ghost Sprint Auto-Archive ✅

**Status:** DONE (pré-implémenté)

`cto_bridge/ghost_sprint_archiver.py` était déjà présent (commit `18d3077`). Les 7 tests TDD passent GREEN :

- `_is_phantom` : détecte correctement les sprints fantômes (status planned/todo + >30j + 0 stories actives)
- `archive_ghost_sprints` : appelle PUT /sprints/{id}/status pour archiver, ignore les sprints récents ou actifs

```
7 passed in 0.31s
```

Aucun commit supplémentaire nécessaire — arbre propre.
